pandas/tests/scripts/test_validate_docstrings.py

```diff
@@ -0,0 +1,344 @@
+import os
+import sys
+
+import numpy as np
+import pytest
+
+import pandas.util._test_decorators as td
+
+
+class GoodDocStrings(object):
+    """
+    Collection of good docstrings - be sure to update the tests as new
+    examples are added here
```

Comment:
Maybe make this class docstring also an example to test? (to be sure that class docstrings also get checked correctly)